### PR TITLE
Initial commit for version 1.0.0

### PR DESCRIPTION
**Pull request by heinzer**
## Description

Made the JSON Schema Form into standalone library containing the following features:
- Checkboxes
- Checkboxes with Children
- Dropdowns
- Radio Buttons
- Dropdowns with Children
- Integer Inputs
- Sections
- Text Inputs
- Text Inputs (Secured)

## Breaking Changes

None.

## Internal Tracking Number

S-19271

## PR Checklist
_All items should be done and checked before merging._
- [x] **Title:** Provide a very brief, general summary.
- [x] **Details:** Fill out the _Description_, _Breaking Changes_, _3rd Party Dependency Changes_, and _Internal Tracking Number_ sections. If there's nothing for a given section, write "None".
- [x] **Labels:** Add one or more labels.
- [x] **Run Unit Tests:** Locally run the Karma unit tests for this project before merging this PR.
- [x] **Run Lint:** Lint the project before merging this PR. 
- [x] **Changelog:** If any code change in this PR will run in production, add an entry to the "Unreleased" section of the `CHANGELOG.md` file.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Changelog
+All notable changes to this project will be documented in this file. The changes are grouped by the date (ISO-8601) and the package version the have been added to. The `Unreleased` section keeps track of upcoming changes.
```
Comment (compeek) [inline CHANGELOG.md]: the -> they

File: README.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+```
+
+### Front End Project Development
+#### Prerequisites
+
+1. To use the front end functionality of this library, your project must use:
+   - Angular
+   - [Node.js](https://nodejs.org/en/download/)
+
+2. [Install the JSON Schema Form]() into your project
```
Comment (compeek) [inline README.md]: This looks like a link that's missing the URL.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+3. Install icons into your front end Angular project
+   1. Open your project's angular.json file. Find the `"assets"` key. Pick a directory to store these images (ex: `./src/assets`).
+   2. Copy the directory `./projects/sfl-launcher/src/assets/sfl-images` into your project's assets directory.
```
Comment (compeek) [inline README.md]: I'm pretty sure you could configure Angular to copy the folder directory out of node_modules, couldn't you? That would be a lot better than manually copying the folder and having to remember to update it when updating the library.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+#### UI Component Configuration
+1. Import the SFLModule into your project.
+```
+import { SFLModule } from '@cleo-cic/sfl';
```
Comment (compeek) [inline README.md]: I forget where we landed on this. Are we counting on using `@cleo-cic` now or is this old?

File: README.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+   (formHeightChange)="onFormHeightChange($event)">
+  </sfl-component>
+```
+
+### Back End Project Development
+#### Prerequisites
+1. To use the back end functionality of this library, it is recommended your project use:
+   - [Node.js](https://nodejs.org/en/download/)
+
+2. [Install JSON Schema Form Validation]()
```
Comment (compeek) [inline README.md]: link with no URL

File: package.json [hunk 6]
```diff
@@ -0,0 +1,55 @@
+{
+  "name": "schema-form-library",
```
Comment (compeek) [inline package.json]: Should we rename this? This is really the non-public package so it probably doesn't matter too much, but nonetheless it seems like we should use the new name.

File: README.md (hunk 4 above)
Comment (compeek) [reply README.md]: And regardless shouldn't `sfl` be `ngx-json-schema-form`?

File: projects/sfl-validation/package.json [hunk 7]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "@cleo-cic/sfl-validation",
```
Comment (compeek) [inline projects/sfl-validation/package.json]: What is the actual `npm` package name going to be for the validation part? I assume it's not `sfl-validation` anymore., so this probably needs to be changed.

File: projects/sfl/package.json [hunk 8]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@cleo-cic/sfl",
```
Comment (compeek) [inline projects/sfl/package.json]: This needs to be changed to match the `npm` package name, right?

File: README.md [hunk 9]
```diff
@@ -0,0 +1,134 @@
...
+      - If this is an edit case
+      - If sections are collapsible
+      - If there are dividers between sections
+   - Inject the SFLDataItemService into your constructor and use the `getFormDataItems()` method to transform the JSON 7 schema and corresponding values into an array of FormDataItems. FormDataItems are the data model that the SFL understands and uses to generate the angular forms.
+   - Create a Submit button in your component. Listen to the `disableSubmit` event emitted from the JSON Schema Form and disable your submit button.
+   - Create a ViewChild property in your component to reference your SFLComponent. Use this property to get the submitted form values by calling `this.schemaFormComponent.getFormValues();`
+   - [Optional] Listen to the `formHeightChange` event emitted from the JSON Schema Form.
+   
+```
+import { FormDataItem, SFLComponent, SFLConfig, SFLDataItemService } from '@cleo-cic/sfl';
```
Comment (compeek) [inline README.md]: same here with both the org name and package name

File: README.md (hunk 2 above)
Comment (heinzer) [reply README.md]: I want to create a installation new section once we have a scope to publish to on npm. But I'll remove the markdown for now.

Comment (compeek) [reply README.md]: Sure, if you're not worried about it for now, no problem. Just wanted to point it out.

File: projects/sfl-validation/package.json (hunk 7 above)
Comment (heinzer) [reply projects/sfl-validation/package.json]: We don't have a scope decided yet, but I'll rename it to ngx-json-schema-form-validation for now.

File: projects/sfl/package.json (hunk 8 above)
Comment (heinzer) [reply projects/sfl/package.json]: Similar to the other comment, we don't have a npm scope yet, so I'll name this ngx-json-schema-form for now.

File: package.json (hunk 6 above)
Comment (heinzer) [reply package.json]: Done!

File: README.md (hunk 3 above)
Comment (heinzer) [reply README.md]: Yeah, that item is definitely on our wishlist to tackle in the future.

File: projects/jsf/README.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
+# SchemaFormLibrary
```
Comment (compeek) [inline projects/jsf/README.md]: This is still the old name, if it matters.

**Review by compeek — APPROVED** [review]
Looks good best I can tell!

**Pull request merged**